fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/api/event/player/AttackBlockCallback.java

```diff
@@ -30,12 +30,19 @@
  * Callback for left-clicking ("attacking") a block.
  * Is hooked in before the spectator check, so make sure to check for the player's game mode as well!
  *
- * <p>Upon return:
- * <ul><li>SUCCESS cancels further processing and, on the client, sends a packet to the server.
- * <li>PASS falls back to further processing.
- * <li>FAIL cancels further processing and does not send a packet to the server.</ul>
+ * <p>On the client-side, the return values have the following meaning:
```

Comment:
```suggestion
 * <p>On the logical client, the return values have the following meaning:
```
I think we use these terms in other places in FAPI too